### PR TITLE
Refactor Shop Display

### PR DESCRIPTION
**Pull request by SerperiorBae**
- Just A General Improvement to help this one match Wisps Colour Scheme. I think it's pretty cute and I don't know about you guys but I like it.
- Preview: http://prntscr.com/bm4r18
- Asked a small handful of users and they seemed to like it.

### CODE REVIEW

File: chat-plugins/credits.js [hunk 1]
```diff
@@ -3,6 +3,12 @@
 const fs = require('fs');
 
 let shopTitle = 'Wisp Credit Shop';
+let shop-table = 'background-color: #ffc775 ; border: #c13a00 solid 1px ; color: #ff560e ; border-collapse: collapse ; padding: 5px ; width: 100%';
```
Comment (panpawn) [inline chat-plugins/credits.js]: consts with uppercase and underscores would look nicer aesthetically

**Comment by MasterFloat** [comment]
I did a shop design just now, so this can be closed.

**Comment by panpawn** [comment]
@MasterFloat  - this is the credits shop, which is apparently different from the regular shop.

If anything, we can probably make the CSS use the same classes we use for the regular CSS.

**Comment by MasterFloat** [comment]
@panpawn - Sundar ( an owner of marketplace ) asked me to make the CSS of creditshop the same as shop.

**Pull request closed without merging**